std/process.d

```diff
@@ -3122,10 +3122,28 @@ private auto executeImpl(alias pipeFunc, Cmd, ExtraPipeFuncArgs...)(
     auto r3 = executeShell("exit 123");
     assert(r3.status == 123);
     assert(r3.output.empty);
-    auto r4 = executeShell("echo stderr test, please ignore 1>&2",
-        null, Config.stderrPassThrough);
-    assert(r4.status == 0);
-    assert(r4.output.empty);
+}
+
+unittest
+{
+    // Temporarily disable output to stderr so as to not spam the build log.
+    import std.stdio : stderr;
+    import std.typecons : Tuple;
+    import std.file : readText;
+    Tuple!(int, "status", string, "output") r;
+    auto tmpname = uniqueTempPath;
+    auto t = stderr;
+    // Open a new scope to minimize code ran with stderr redirected.
+    {
+        stderr.open(tmpname, "w");
+        scope(exit) stderr = t;
+        r = executeShell("echo D rox. 1>&2", null, Config.stderrPassThrough);
+    }
+    assert(r.status == 0);
+    assert(r.output.empty);
+    auto witness = readText(tmpname);
+    import std.ascii : newline;
+    assert(witness == "D rox." ~ newline, "'" ~ witness ~ "'");
```

Comment:
It looks like this is still failing.

`text([str])` or `format("%(%s%)", str.only)` is good for stringifying strings with special characters, as they will show up quoted and with escapes.

The problem might be the space before the `1>&2`, which I think will be literal on Windows.

Reply:
Looks like it worked with `echo D rox>&2`.